text/1004-transactions-api.rst

```diff
@@ -391,6 +391,53 @@ The ``RetryOptions`` signature:
         ): RetryOptions;
     }
 
+Subtransactions API
+'''''''''''''''''''
+
+Subtransaction API is only accessible on the transaction object:
+
+.. code-block:: typescript
+
+    class Transaction {
+        async subtransaction<T>(action: TransactionBlock): Promise<T>;
+    }
+
+Example using ``subtransaction``:
```

Comment:
For JS, maybe the name should be `subTransaction` (plays better with `tryTransaction`)?  Although Postgres calls it `subtransansaction` in their APIs and this is a single English word. @elprans ?

Reply:
two cents:

> plays better with tryTransaction

The motivation for this seems to be purely aesthetic and overlooks the function part of casing conventions. It doesn't take into account the camel casing pattern of making the first letter of words capital. This makes an arbitrary letter capital. It seems like this decision would make it harder for people to guess the correct spelling.

Reply:
Good point, Frederick.